CHANGES/9472.bugfix

```diff
@@ -0,0 +1,6 @@
+Restored the functionality of specifying basic-auth parameters in a remote's URL.
+
+NOTE: it's much better to specify username/pwd explcitly on the Remote, rather
+than relying on embedding them in the URL. This fix will probably be deprecated in
+the future.
+(backported from #9464)
```

Comment:
I'm guessing you ran into conflicts and had to do it manually - could you copy this format? 

https://github.com/pulp/pulp_rpm/commit/9ba3b5757ea23384f94849e470a61e151476a374